bin/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name
+  attr_reader :publish_date
```

Comment:
You don't need to add this line, due this is already specified in the super class.